.github/workflows/auto-publish.yml

```
@@ -1,72 +1,71 @@
+        run: |
+          TOKEN=$(printf '%s' "$NODE_AUTH_TOKEN" | tr -d '[:space:]')
+          npm config set registry https://registry.npmjs.org/
+          npm config set //registry.npmjs.org/:_authToken "$TOKEN"
+          npm publish --access=public
         env:
-          # 使用NPM令牌进行身份验证
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        working-directory: packages/theme-mobile/dist
 
       - name: Release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)